src/parser/parser_utils.js

```diff
@@ -146,4 +146,38 @@ export class ParserUtils {
     }
     return hours + minutes + seconds;
   }
+
+  /**
+   * Splits an Array of ads into an Array of Arrays of ads.
+   * Each subarray contains either one ad or multiple ads (an AdPod)
+   * @param  {Array} ads - An Array of ads to split
+   * @return {Array}
+   */
+  splitVAST(ads) {
+    const splittedVAST = [];
+    let lastAdPod = null;
+
+    ads.forEach((ad, i) => {
+      if (ad.sequence) {
+        ad.sequence = parseInt(ad.sequence);
```

Comment:
Always use the 2nd parameter of parseInt to specify the base, here 10 `ad.sequence = parseInt(ad.sequence, 10);`